kjango/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for kjango project.
+
+Generated by 'django-admin startproject' using Django 1.11.12.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+# from decouple import config
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '$)l5l15j#nd4ln3z)+6*hj&2gmh#hknxs*%c0s7sjvzispe*1('
+# SECRET_KEY = config('SECRET_KEY')
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+# DEBUG = config('DEBUG', cast=bool)
```

Comment:
you want to put the values for `SECRET_KEY` and `DEBUG` in a `.env`file.  I looks like you tried to `from decouple import config` but i don't see `python-decouple` as an installed pkg in your `pipfile`